arch/arm64/core/mmu.c

```diff
@@ -4,6 +4,8 @@
  *
  * Copyright (c) 2021 BayLibre, SAS
  *
+ * Copyright 2022 NXP
+ *
```

Comment:
This is not a change big enough to justify a copyright change.

Reply:
okay, will remove it.